UANodeManager.cs

```diff
@@ -173,7 +236,10 @@ public override void CreateAddressSpace(IDictionary<NodeId, IList<IReference>> e
                 {
                     externalReferences[ObjectIds.ObjectsFolder] = references = new List<IReference>();
                 }
-                
+
+                ImportFromXml();

```

Comment:
Perhaps put your  if (!_useWotNodesetBinary) here?

Reply:
Yes, I'll add a check to both xml and binary here.